resources/Materials/Examples/StandardSurface/standard_surface_wood_tiled.mtlx

```diff
@@ -1,8 +1,12 @@
 <?xml version="1.0"?>
 <materialx version="1.38" colorspace="lin_rec709">
   <nodegraph name="NG_wood1" fileprefix="../../../Images/">
+    <dot name="color_filename" type="filename">
+      <input name="in" type="filename" value="wood_color.jpg" />
+      <input name="note" type="string" value="Checking for spurious samplers" />
+    </dot>
     <tiledimage name="image_color" type="color3">
-      <input name="file" type="filename" value="wood_color.jpg" colorspace="srgb_texture" />
+      <input name="file" type="filename" nodename="color_filename" colorspace="srgb_texture" />
```

Comment:
Sufficient to trigger the issue fixed in #1152. This will not generate valid GLSL code unless dot elision is active for filenames.